src/bw_api.py

```diff
@@ -48,11 +48,57 @@ def get_grading_run_status(cid, aid, run_id):
 	:return: a status string if successful, or None otherwise.
 	"""
 	try:
-		resp = requests.get(url="%s/grading_run/%s/%s/%s" % (BROADWAY_API_URL, cid, aid, run_id), headers=HEADERS)
-		return resp.json()["data"]["student_jobs_state"][0]
-	except requests.exceptions.RequestException:
+		resp = requests.get(url="%s/grading_run_status/%s/%s" % (BROADWAY_API_URL, cid, run_id), headers=HEADERS)
+		student_jobs_dict = resp.json()["data"]["student_jobs"]
+		return student_jobs_dict.values().next()
```

Comment:
I don't recall there is a `next` method for `dict_values`. Is this method added by the json library or something?

Reply:
`dict.values()` should return an iterator in Python3, so `next()` should return the first item in the iterator.

Reply:
Nvm, you're right. It's not an iterator.